scripts/migrate-black.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+# check out every commit added by the current branch, blackify them,
+# and generate diffs to reconstruct the original commits, but then
+# blackified
+import logging
+import os
+import sys
+from subprocess import check_output, run, Popen, PIPE
+
+
+def git(*args):
```

Comment:
`mypy` CI is failing - Can you please type annotate the code since we have very high type coverage elsewhere.

Reply:
done